src/loaders/ObjectLoader.js

```diff
@@ -401,22 +402,41 @@ Object.assign( ObjectLoader.prototype, {
 
 			for ( var i = 0, l = json.length; i < l; i ++ ) {
 
-				var data = json[ i ];
+				var data = json[ i ], texture;
 
-				if ( data.image === undefined ) {
+				if ( data.images !== undefined ) {
```

Comment:
Check for .images property to determine if this is a CubeTexture.

We can't check `.type` as the type property refers to something else for Textures